app/services/store_policy_resolver.py

```
@@ -0,0 +1,641 @@
+                collective_goal_target=collective_goal_target,
+                collective_goal_expires_at=collective_goal_expires_at,
+                policy_uuid=policy_uuid,
+                class_id=class_id,
+                created_at=created_at,
+            )
+
+        except PolicyParseError:
+            raise
+        except PolicyValidationError:
+            raise
+        except Exception as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)